changes/2472.misc.rst

```diff
@@ -0,0 +1 @@
+Updated CI to use shared workflows from ``.github`` repository.
```

Comment:
The ticket number for this file seems... wrong

Reply:
whoops....dont even know where that number came from....